src/SearchBar.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+
+function SearchBar(props) {
+    const [query, setQuery] = useState("");
+
+    return (
+        // once the user has pressed enter or the search button, it will then allow all pictures to render to the page
+        <form className="form" onSubmit={
```

Comment:
This looks good. You could add a handle submit function so that after submit your search bar defaults back to 'placeholder' and clears the users input after submit